packages/application-generic/src/usecases/add-job/merge-or-create-digest.usecase.ts

```diff
@@ -213,4 +219,23 @@ export class MergeOrCreateDigest {
       job._organizationId
     );
   }
+  private async digestSkippedExecutionDetails(
+    job: JobEntity,
+    filtered: boolean
+  ): Promise<void> {
+    const metadata = CreateExecutionDetailsCommand.getExecutionLogMetadata();
+    await this.executionLogQueueService.add(
+      metadata._id,
+      CreateExecutionDetailsCommand.create({
+        ...metadata,
+        ...CreateExecutionDetailsCommand.getDetailsFromJob(job),
+        detail: filtered ? DetailEnum.FILTER_STEPS : DetailEnum.DIGEST_SKIPPED,
+        source: ExecutionDetailsSourceEnum.INTERNAL,
+        status: ExecutionDetailsStatusEnum.SUCCESS,
+        isTest: false,
+        isRetry: false,
+      }),
+      job._organizationId
+    );
```

Comment:
Show execution details for a skipped digest - for backoff as well. We only showed 'step created'

Reply:
Great addition!